### PR TITLE
Use relative links, add prereqs to Helm README

### LINKED ISSUE #28: [DOC]: Document the JSON schema for extracted content
### How would you describe the priority of this documentation request

Significant improvement

### Describe the future/missing documentation

nv-ingest produces JSON formatted extraction results. Users need documentation on how to read it

### LINKED ISSUE #31: [DOC]: Include minimum hardware requirements in the Helm README
### How would you describe the priority of this documentation request

None

### Please provide a link or source to the relevant docs

github.com/nvidia/nv-ingest

### Describe the problems in the documentation

The helm chart deployment needs to note the minimum hardware and software requirements necessary.

### (Optional) Propose a correction or improvement

_No response_

### PR DESCRIPTION
**Pull request by randerzander**
Closes #28, #29, #31, #34, #36, #40, #41, #45, #46, #47, #50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,31 +68,31 @@ This example demonstrates how to use the provided [docker-compose.yaml](docker-c
...
 
-To start all services:
+4. To start all services:
 `docker compose up`
 
-Please note, NIM containers on their first startup can take 10-15 minutes to pull and fully load models. Also note that by default we have [configured log levels to be verbose](docker-compose.yaml#L31) so it's possible to observe service startup proceeding. You will notice _many_ log messages. You can control this on a per service level via each service's environment variables.
+Please note, NIM containers on their first startup can take 10-15 minutes to pull and fully load models. Also note that by default we have [configured log levels to be verbose](docker-compose.yaml#L27) so it's possible to observe service startup proceeding. You will notice _many_ log messages. You can turn off verbose logging for by configuring `NIM_TRITON_LOG_VERBOSE=0` for each NIM in [docker-compose.yaml](docker-compose.yaml).
```
Comment (jdye64) [inline README.md]: "verbose logging for by configuring" -> "verbose logging by configuring"

File: README.md [hunk 2]
```diff
@@ -105,12 +108,23 @@ When all services have fully started, `nvidia-smi` should show processes like th
 |    3   N/A  N/A   1355929      C   ...s/python/triton_python_backend_stub      424MiB |
 |    3   N/A  N/A   1373202      C   tritonserver                                414MiB |
 +---------------------------------------------------------------------------------------+
-
 ```
 If it's taking > 1m for `nvidia-smi` to return, it's likely the bus is still busy setting up the models.
 
 Once it completes normally (less than a few seconds), the NIM models are ready.
 
+You can observe the started containers with `docker ps`:
```
Comment (jdye64) [inline README.md:116]: Shouldn't the NIM containers be shown in this output?

**Review by jdye64 — CHANGES_REQUESTED** [review]
Trivial changes but otherwise looks good

File: README.md (hunk 2 above)
Comment (randerzander) [reply README.md:116]: good catch. Missed those in the output.

**Review by jdye64 — APPROVED** [review]

**Pull request merged**